### ISSUE TITLE
Is it possible to use it with Fydeos?

### ISSUE THREAD

**Issue by bybenjamin** [sebanc/brioche#45]
What do I need to do to use it with Fydeos?

**Comment by dwemerlord667** [comment]
Yeah works fine with the latest version of FydeOS - FydeOS is just a modified version of ChromiumOS so any software designed for ChromeOS should work with FydeOS

**Comment by bybenjamin** [comment]
i will try now

**Comment by bybenjamin** [comment]
it didn't work
To enable SSH, run: apt install openssh-server
No default root or user password are set by LXC.
lxc-start: kali: lxccontainer.c: wait_on_daemonized_start: 868 Received container state "ABORTING" instead of "RUNNING"
lxc-start: kali: tools/lxc_start.c: main: 308 The container failed to start
lxc-start: kali: tools/lxc_start.c: main: 311 To get more details, run the container in foreground mode
lxc-start: kali: tools/lxc_start.c: main: 313 Additional information can be obtained by setting the --logfile and --logpriority options
Permission denied - Failed to rexec as memfd
free(): invalid pointer
